frontend/src/components/Public/CrossingListItem/PublicCrossingListItem.js

```diff
@@ -28,80 +35,77 @@ class PublicCrossingListItem extends React.Component {
-          </div>
-        </div>
-        <div className="CrossingListItemFlexContainer">
-          <div className="CrossingListItemFlexItem">
-            <div className="ControlLabel">
-              Status: {statusConstants.statusNames[crossing.latestStatusId]}
+            <div>
+              <Location crossing={crossing} />
+              <CrossingCommunityList crossing={crossing} />
             </div>
           </div>
-
-          {show.includes('reason') ? (
-            <div className="CrossingListItemFlexItem">
-              <div className="ControlLabelContainer">
-                <div className="ControlLabel">Reason</div>
-              </div>
+          <div className="PublicCrossingListItem__datetime">
+            <div className="PublicCrossingListItem__datetime-date">
+                <Date date={createdAt} />
             </div>
-          ) : (
-            <div className="CrossingListItemFlexItem--spacer" />
-          )}
-          <div className="CrossingListItemFlexItem">
-            <div className="ControlLabel">Notes to the public</div>
-          </div>
-        </div>
-
-        {show.includes('duration') ? (
-          <div className="CrossingListItemFlexContainer">
-            <div className="CrossingListItemFlexItem--spacer" />
-            <div className="CrossingListItemFlexItem">
-              <div className="ControlLabelContainer">
-                <div className="ControlLabel">Duration</div>
-              </div>
+            <div>
+              <Hour date={createdAt} />
             </div>
           </div>
-        ) : (
-          <div className="CrossingListItemFlexItem--spacer" />
-        )}
+        </div>
+        <div className="PublicCrossingListItem__details">
+          {show.includes('reason') && (
+            <DetailsItem title="Reason">
+              {/* TODO: Add actual reason */}
+              Blah reason
+            </DetailsItem>
+          )}
+          {show.includes('reopen') && (
+            <DetailsItem title="Reason">
+              {/* TODO: Add actual reopen */}
+              Blah minutes
+            </DetailsItem>
+          )}
+          {show.includes('notes') && (
+            <DetailsItem title="Notes">
+              {/* TODO: Add actual reopen */}
+              {crossing.statusUpdateByLatestStatusUpdateId.notes}
+            </DetailsItem>
+          )}
+        </div>
       </div>
     );
 
     if (!this.props.cqClassName) {
       return (
         <ContainerQuery query={containerQuery}>
           {params => (
-            <div className={classnames(params, 'CrossingListItem')}>
-              {CrossingListItemJSX}
-            </div>
+            <div className={classnames(params)}>{CrossingListItemJSX}</div>
           )}
         </ContainerQuery>
       );
     }
 
     return (
-      <div className={classnames(this.props.cqClassName, 'CrossingListItem')}>
+      <div className={classnames(this.props.cqClassName)}>
```

Comment:
I know you mentioned removing the containerquery stuff from the DashboardCrossingListItem, once you have a pattern established for that it'd be cool to use it for this too.

For now, since we don't have a public list view so containerquery for it might be overkill.